REST_API_REFERENCE.md

```diff
@@ -0,0 +1,74 @@
+# REST API Reference (v1)
+
+1. [API Endpoint](#api-endpoint)
+2. [Analytics](#analytics)
+3. [Application](#application)
+4. [Content](#content)
+5. [Custom Project](#custom-project)
+
+
+## API Endpoint
+
+Base URL: `http://elimu.ai/rest/v1`
+
+
+## Analytics
+
+TODO
+
+
+## Application
+
+TODO
+
+
+## Content
+
+TODO
+
+
+## Custom Project
+
+### License
+
+#### Read
+
+URL: `/project/license`
+
+Example request:
+```
+HTTP GET
+/project/license?licenseEmail=info@elimu.ai&licenseNumber=bddf-d8f4-2adf-a365
```

Comment:
The API description looks better if parameters are specified separately with its expected data type, for instance:

licenseEmail: String
licenseNumber: String
....

Reply:
Like this?: https://github.com/elimu-ai/webapp/blob/%23537-Add-REST-API-documentation-for-custom-project/REST_API_REFERENCE.md

Reply:
Yes, I think it's clear enough now.